bitnami/etcd/3.5/debian-11/rootfs/opt/bitnami/scripts/libetcd.sh

```diff
@@ -664,6 +664,8 @@ etcd_initialize() {
             if is_boolean_yes "$ETCD_DISABLE_PRESTOP"; then
```

Comment:
I think there is a misunderstanding on where the code needs to be placed. I meant this:

```suggestion
            if is_boolean_yes "$ETCD_DISABLE_PRESTOP"; then
                info "The member will try to join the cluster by it's own"
                export ETCD_INITIAL_CLUSTER_STATE=existing
            elif ! is_healthy_etcd_cluster; then
                warn "Cluster not responding!"
                if is_boolean_yes "$ETCD_DISASTER_RECOVERY"; then
                    latest_snapshot_file="$(find /snapshots/ -maxdepth 1 -type f -name 'db-*' | sort | tail -n 1)"
```

So removing this duplicate code:

```
            elif ! is_healthy_etcd_cluster; then
                warn "Cluster not responding!"
            fi
            member_id="$(get_member_id)"
            if ! is_healthy_etcd_cluster; then
                warn "Cluster not responding!"
```

Reply:
sincere apologies for the churn :(.  I've pushed another commit after testing with your suggested change, worked great!  let me know if this looks ok now?